frontend/app/src/types/asset/index.ts

```diff
@@ -102,7 +103,8 @@ export const CustomAssets = CollectionCommonFields.extend({
 });
 export type CustomAssets = z.infer<typeof CustomAssets>;
 
-export interface CustomAssetPagination extends ApiPagination<CustomAsset> {
+export interface CustomAssetRequestPayload
+  extends PaginationRequestPayload<CustomAsset> {
```

Comment:
can you check if ApiPagination/PaginationRequestPayload are overlapping?

Reply:
checking

Reply:
yes, they do, but these 2 are extras in `PaginationRequestPayload`

```typescript
readonly ignoreCache?: boolean;
readonly onlyCache?: boolean;
```

Reply:
and these are not required in `PaginationRequestPayload`

```typescriopt
readonly orderByAttributes?: (keyof T)[];
readonly ascending?: boolean[];
```